.github/workflows/bundle-size.yml

```diff
@@ -0,0 +1,23 @@
+name: Compressed Size
+
+on:
+    pull_request:
+    workflow_dispatch:
+
+jobs:
+    build:
+        name: Bundle size check
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use desired version of NodeJS
+              uses: actions/setup-node@v2
+              with:
+                  node-version-file: '.nvmrc'
+                  cache: npm
+            - uses: preactjs/compressed-size-action@v2
+              with:
+                  repo-token: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
I don't see `GITHUB_TOKEN` secret under repository secrets. Have you added it? 

If you haven't, try creating it with access only to public repositories. I haven't tested it, but let me know how it goes :) 

<img width="806" alt="Screenshot 2022-09-22 at 9 57 32 PM" src="https://user-images.githubusercontent.com/1161872/191801808-94d1b378-835e-48eb-af68-9bd314d65cc6.png">

Reply:
I have created a token with public repo access and added it to the repo secrets.